File: JS/encrypt.js
/********************************

Use: "message".encrypt()
=> 😝😅😒😒😄😍😅
********************************/
String.prototype.encrypt = function () {
	const emojis ='😄,😃,😀,😊,😅,😉,😍,😘,😚,😗,😙,😜,😝,😛,😳,😁,😔,😌,😒,😞,😣,😢,😂,😭,😎,😈'.split(",")

	const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")
	let out = ""
	for (let o of this.toUpperCase()) {
		out += emojis[alphabet.indexOf(o)]
	}
	return out;
};

module.exports = text => text.encrypt();

File: Python3/encrypt.py
mydict = {"A":"😄","B":"😃","C":"😀","D":"😊","E":"😅","F":"😉","G":"😍","H":"😘","I":"😚","J":"😗","K": "😙","L":"😜","M":"😝","N":"😛","O":"😳","P":"😁","Q":"😔","R":"😌","S":"😒","T":"😞","U": "😣","V":"😢","W":"😂","X":"😭","Y":"😎","Z":"😈"}

def encrypt(message):
  """ Use : encrypt("MESSAGE")
=> '😝😅😒😒😄😍😅'
  """
  message = message.upper()
  output = ""
  for character in message:
    output += mydict[str(character)]
  return output
